src/index.tsx

```diff
@@ -41,12 +41,19 @@ epConfig.supportedLocales.forEach((locale) => {
   locales[locale.value] = { ...localeMessages, ...debugMessages };
 });
 
+if (process.env.NODE_ENV !== 'production') {
```

Comment:
We may re-visit this later. I'm wondering if this is enough. This would mean we're still going to surface accessibility scans in dev mode too and it's not toggled.